### PR TITLE
Build HTML tables and guides when building product specific content

### PR DESCRIPTION
**Pull request by mpreisler**
`make -j 4 rhel6` now also builds HTML guides and tables. The content only target is now `make -j 4 rhel6-content`.

This makes cmake more similar to the old system.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -30,19 +30,29 @@
...
     `$ make -j4 rhel6`
 
+  * Other targets only for one specific product:
+
+    `$ cd scap-security-guide/`
+    `$ cd build/`
+    `$ cmake ../`
+    `$ make -j4 rhel6-content  # SCAP XML files for RHEL6
+    `$ make -j4 rhel6-guides  # HTML guides for RHEL6
+    `$ make -j4 rhel6-tables  # HTML tables for RHEL6
```
Comment (shawndwells) [inline BUILD.md]: need closing ticks

Comment (mpreisler) [inline BUILD.md]: just noticed this as well, fixed

**Comment by shawndwells** [comment]
works for me locally. will watch for jenkins to finish up and merge.

**Comment by mpreisler** [comment]
The second jenkins run failed because of an issue I outlined here: https://github.com/OpenSCAP/openscap/pull/687

I will work around it in cmake in a separate PR but I don't think it should block this PR. The jenkins run before that worked fine and the last commit just added missing ticks.

**Comment by mpreisler** [comment]
#1724 fixes the jenkins issue for good.

**Comment by redhatrises** [comment]
Ack.

**Pull request merged**